### PR TITLE
Fixed compiled css validation

### PR DESCRIPTION
**Pull request by passionInfinite**
Fixed the compiled CSS validation and added public/fonts in .gitignore to ignore this files.

### CODE REVIEW

**Comment by passionInfinite** [comment]
The verified output screenshot:
[ScreenShot](https://prnt.sc/jxbhtb)

File: app/.gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@ composer.lock
 Homestead.json
 Homestead.yaml
 .env
+/public/fonts
```
Comment (joshi1983) [inline app/.gitignore]: This change probably isn't needed if you get latest on your master branch and rebase.
Specifically this commit is what you'd want: https://github.com/hhaccessibility/hhaccessibility.github.io/commit/62885f6e9a6281ea8bc25af70745571e3750aa2a

Your master in your fork(https://github.com/passionInfinite/hhaccessibility.github.io) is behind master in the main repository(https://github.com/hhaccessibility/hhaccessibility.github.io/) by about 50 commits which included a change to .gitignore.  

https://github.com/hhaccessibility/hhaccessibility.github.io/wiki/git-commands has some command sequences that could help if you have some trouble.

File: app/resources/assets/sass/_location_management.scss [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 	input[type="text"], select {
 		width: 100%;
-		box-size: border-box;
+		box-sizing: border-box;
```
Comment (joshi1983) [inline app/resources/assets/sass/_location_management.scss]: good catch.

File: app/.gitignore (hunk 1 above)
Comment (passionInfinite) [reply app/.gitignore]: Yea! I synced my branches merges this time. From Next time I will take care of it :+1:

Comment (joshi1983) [reply app/.gitignore]: ok.  I'll merge this and then merge a commit that removes this line so it isn't forgotten.  It is a pretty small detail but I just don't want to waste the discovery by forgetting about it.

**Pull request merged**